DotaStats/Modules/MatchesModule/Presenter/MatchesModulePresenter.swift

```diff
@@ -0,0 +1,134 @@
+import Foundation
+
+protocol MatchesModuleInput: AnyObject {}
+
+protocol MatchesModuleOutput: AnyObject {
+    func matchesModule(_ module: MatchesModuleInput, didSelectPlayer matchId: Int)
+}
+
+final class MatchesModulePresenter {
+    weak var view: MatchesModuleViewInput?
+    private let matchesService: MatchesService
+    private var matches: [MatchCollectionPresenterData] = []
+    let output: MatchesModuleOutput
+    
+    required init(matchesService: MatchesService,
+                  output: MatchesModuleOutput)
+    {
+        self.matchesService = matchesService
+        self.output = output
+        self.state = .none
+    }
+    
+    private var state: MatchesModulePresenterState {
+        didSet {
+            oldValue.token?.cancel()
+            switch state {
+            case .result(let requestResult):
+                switch requestResult {
+                case .success(var matches):
+                    matches.sort { $0.startTime < $1.startTime }
+                    convertMatches(matches: matches)
```

Comment:
Если чуть изменить сигнатуру функции, то может получиться так
```suggestion
                    convert(matches)
```
Ну и по имени кажется, что функция должна что-то вернуть. Если в функции как-то хендлятся матчи, то лучше ее переименовать во что-то типа `handle(matches)`